extensions/distributed-tracing.md

```diff
@@ -24,25 +24,31 @@ Prometheus are built.
 - Constraints
   - OPTIONAL
 
-## HTTP encoding
+## Using the Distributed Tracing Extension
 
-To integrate with existing tracing libraries, the Distributed Tracing attributes
```

Comment:
My 2p and take it for what its worth. I don't see the update of this section bringing clarity the linked specification doesn't already cover. Maybe just underscore that the headers themselves are only the context of the trace.

Other event attributes may be helpful to add to trace data (or even some as log correlations or metrics dimensions), but specifically how to do log correlation, metrics or tracing is out of scope for this spec.

I'm aware that mentioning Zipkin may not be ok here, but if you did there's a lot of existing art with messaging tracing. Your call.

Reply:
> tracing is out of scope for this spec.

That's what we're trying to establish here :smile: The reason of this PR is exactly because somebody thought that the goal of this spec extension is to enable tracing.

Reply:
I see. If the goal is to enable tracing, it seems limiting to only discuss the tracecontext spec. many years of tools, thousands of users are familiar with B3. You might consider linking to this also, if adoption is a concern. https://github.com/openzipkin/b3-propagation

Reply:
This is another reason why i feel we should not touch the distributed tracing argument at all, otherwise we have a clear overlap with what openzipkin/opentelemetry/w3c is doing

Reply:
I agree that specifically how to do log correlation, metrics, and tracing are out of scope of this extension. That does not imply, however, that trace context propagation is out of scope. Since this extension is modeled after the W3C trace propagation format it seems that trace context propagation must be in-scope here. Nor do I think that adding a trace propagation mechanism for events conflicts with those other projects especially given that they are unlikely to define trace propagation standards either for cloudevents or for all possible eventing protocols.